includes/class-theme-base.php

```diff
@@ -1496,7 +1559,73 @@ function the_pagination_html( $args = array() ) {
 	 */
 	function get_pagination_html( $args = array() ) {
 
-		echo get_the_posts_pagination( $args );
+		get_the_posts_pagination( $args );
 	}
 
+	/**
+	 * Returns the navigation to next/previous set of posts, when applicable.
+	 *
+	 * @since WP 4.1.0
+	 *
+	 * @global WP_Query $wp_query WordPress Query object.
+	 *
+	 * @param array $args {
+	 *     Optional. Default posts navigation arguments. Default empty array.
+	 *
+	 *     @type string $prev_text          Anchor text to display in the previous posts link.
+	 *                                      Default 'Older posts'.
+	 *     @type string $next_text          Anchor text to display in the next posts link.
+	 *                                      Default 'Newer posts'.
+	 *     @type string $screen_reader_text Screen reader text for nav element.
+	 *                                      Default 'Posts navigation'.
+	 * }
+	 *
+	 * @return string Markup for posts links.
+	 */
+	function get_posts_navigation_html( $args = array() ){
+		return get_the_posts_navigation( $args );
+	}
+
+	/**
+ 	 * Outputs the navigation to next/previous set of posts, when applicable.
+	 *
+	 * @param array $args
+	 */
+	function the_posts_navigation_html( $args = array() ){
+		echo wp_kses_post( $this->get_posts_navigation_html( $args ) );
+	}
+
+
+	/**
+	 * Whether the site is being previewed in the Customizer.
+	 *
+	 * @return bool
+	 */
+	function is_customize_preview(){
+		return is_customize_preview();
```

Comment:
@sarutole `is_customize_preview()` uses `global $wp_customize`.  Since one of WPLib's goal is to eliminate a lot of errors around globals, shouldn't we capture `$wp_customize` into a property of `$theme` and use it directly here by saving, setting then restoring the global var?